Assets/Scripts/Managers/HouseManager.cs

```diff
@@ -10,6 +10,7 @@ public enum HouseMode { Explore, Decorate }
 	public static event OnModeChange ModeChanged;
 
 	[SerializeField] private CanvasGroup decorateUI;
+	[SerializeField] private MeshFilter playerModel;
```

Comment:
This is probably better represented by a `MeshRenderer` rather than a `MeshFilter`, and we could set `playerModel.enabled` instead. Also need to discuss what happens if the player places furniture where they are standing (discussion with design?)